scripts/burn-image.sh

```diff
@@ -0,0 +1,218 @@
+#!/bin/bash
+# This script will burn the Raspbian OS Lite Image from the raspberrypi.org server and burn the image to a SD-card.
+# The execution of the script is usually done by technical personnel with the...
+#
+# $ sudo chmod 770 ./burn-image.sh && sudo ./burn-image.sh
+#
+# ...on a computer with a SD-card reader.
+# You have to know what you are doing and we don’t take any responsibility for data loss.
+
+# check whether script runs from a superuser (sudo)
+if [ -z "$(whoami | grep root)" ]
+then
+  echo "Not running as root."
+  echo "Script ended."
+  exit 1    
+fi
+
+
+
+# === GENERAL SETTINGS ===
+
+# set working directory
+workingdir=/tmp/raspbian
+
+# download link to the Raspbian image
+image=https://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2021-05-28/2021-05-07-raspios-buster-armhf-lite.zip
+
+# name of the archive
+archive=raspbian-os-lite.zip
+
+# wget log filename
+wgetlog=wget-raspbian.log
+
+# minimal capacity required on SD-card (0 has no minimum)
+mincap=2000000000
+
+# mount point for boot partition
+mntboot=/media/boot
+
+
+
+# === FUNCTIONS ===
+
+lsstorage() {
+  echo "$(lsblk -e7 | grep -v NAME)" > $1
+  echo "$(sort $1)" > $1
+}
+
+cleanup() {
+  rm -rf $1
```

Comment:
It is not safe to use parameter, better is to hardcode the folder: rm -rf $workingdir. This to prevent a call of 'cleanup /' which essentially destroys your whole system